go.mod

```diff
@@ -6,72 +6,62 @@ require (
-	golang.org/x/crypto v0.0.0-20200115085410-6d4e4cb37c7d // indirect
-	golang.org/x/lint v0.0.0-20191125180803-fdd1cda4f05f // indirect
-	golang.org/x/net v0.0.0-20200114155413-6afb5195e5aa // indirect
+	github.com/prometheus/client_golang v1.5.1
+	github.com/prometheus/procfs v0.0.11 // indirect
+	go.uber.org/zap v1.14.1 // indirect
+	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
+	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
-	golang.org/x/sys v0.0.0-20200116001909-b77594299b42
+	golang.org/x/sys v0.0.0-20200409092240-59c9f1ba88fa
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
-	golang.org/x/tools v0.0.0-20200116062425-473961ec044c // indirect
 	google.golang.org/appengine v1.6.5 // indirect
-	google.golang.org/genproto v0.0.0-20200115191322-ca5a22157cba // indirect
+	google.golang.org/genproto v0.0.0-20200413115906-b5235f65be36 // indirect
 	google.golang.org/grpc v1.26.0
-	gopkg.in/square/go-jose.v2 v2.4.1 // indirect
-	gopkg.in/yaml.v2 v2.2.7 // indirect
-	k8s.io/api v0.17.1
-	k8s.io/apiextensions-apiserver v0.17.1 // indirect
-	k8s.io/apimachinery v0.17.3-beta.0
-	k8s.io/client-go v0.17.1
-	k8s.io/cloud-provider v0.17.1
-	k8s.io/cri-api v0.17.1 // indirect
+	gopkg.in/square/go-jose.v2 v2.5.0 // indirect
+	k8s.io/api v0.18.0
+	k8s.io/apimachinery v0.18.0
+	k8s.io/client-go v0.18.0
+	k8s.io/cloud-provider v0.18.0
+	k8s.io/cri-api v0.18.0 // indirect
 	k8s.io/klog v1.0.0
-	k8s.io/kubectl v0.17.1 // indirect
-	k8s.io/kubernetes v1.17.1
-	k8s.io/utils v0.0.0-20200109141947-94aeca20bf09
+	k8s.io/kubectl v0.18.0 // indirect
+	k8s.io/kubernetes v1.18.0
+	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
 )
 
 replace (
-	k8s.io/api => k8s.io/api v0.17.1
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.1
-	k8s.io/apimachinery => k8s.io/apimachinery v0.17.1
-	k8s.io/apiserver => k8s.io/apiserver v0.17.1
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.17.1
-	k8s.io/client-go => k8s.io/client-go v0.17.1
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.1
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.17.1
-	k8s.io/code-generator => k8s.io/code-generator v0.17.1
-	k8s.io/component-base => k8s.io/component-base v0.17.1
-	k8s.io/cri-api => k8s.io/cri-api v0.17.1
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.17.1
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.17.1
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.17.1
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.17.1
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.17.1
-	k8s.io/kubectl => k8s.io/kubectl v0.17.1
-	k8s.io/kubelet => k8s.io/kubelet v0.17.1
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.17.1
-	k8s.io/metrics => k8s.io/metrics v0.17.1
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.1
+	google.golang.org/grpc => google.golang.org/grpc v1.26.0
```

Comment:
why grpc downgrade is required?

Reply:
There is a breaking change with 1.27.0 or above wrt etcd. This has to be carefully examined and done later.. so the downgrade.